aspnetcore/blazor/get-started.md

```diff
@@ -19,9 +19,9 @@ To get started with Blazor, follow the guidance for your choice of tooling:
 
 # [Visual Studio](#tab/visual-studio)
 
-1. To create Blazor Server apps, install [Visual Studio 2019 version 16.4 or later](https://visualstudio.microsoft.com/vs/preview/) with the **ASP.NET and web development** workload.
+1. To create Blazor Server apps, install [Visual Studio 2019 version 16.4 or later](https://visualstudio.microsoft.com/downloads/) with the **ASP.NET and web development** workload.
```

Comment:
RTM builds should suffice.

Reply:
@pranavkm If we're recommending RTM builds, should the link text recommend 16.5 instead of 16.4? I assume we want customers using the latest stable RTM build.

Reply:
... or perhaps just say generically "install the latest version." These numbers become stale and silly several months later.

> 1. To create Blazor Server apps, install the latest version of \[Visual Studio](https://visualstudio.microsoft.com/downloads/) with the **ASP.NET and web development** workload.